node-operators/networks/full-node.md

```diff
@@ -191,10 +202,13 @@ Next, we need to create a directory to store the binary and data. We'll also set
 
 ```
 mkdir {{ networks.moonbase.node_directory }}
-chmod 0755 {{ networks.moonbase.node_directory }}
+sudo chown -R $(id -u):$(id -g) {{ networks.moonbase.node_directory }}
 chown moonbase_service {{ networks.moonbase.node_directory }}
```

Comment:
this is redundant with the line above, maybe we should standardize on this line instead?  some users will be running as root, this makes it clear they should set the perms for the service user.